Code.gs

```diff
@@ -57,10 +65,14 @@ function GenerateForms() {
 
       // add manager as editor
       associateForm.addEditor(managerEmail);
+      if (viewerEmail != managerEmail && viewerEmail != "") {
+        //associateForm.addViewer(viewerEmail); // viewers aren't supported
+        associateForm.addEditor(viewerEmail);
+      }
 
       // send email to reviewers and cc manager
       for (var j = 0; j < reviewers.length; j++) {
-        email = GmailApp.createDraft(reviewers[j], title, "You are kindly asked to provide feedback via this form.\n\n" + associateForm.getPublishedUrl() + "\n\nResults will be visible only to the direct lead of the reviewed person.\n\n-- \nFederico\n", { cc: managerEmail });
+        email = GmailApp.createDraft(reviewers[j], title, "You are kindly asked to provide feedback via this form.\n\n" + associateForm.getPublishedUrl() + "\n\nResults will be visible only to the direct lead of the reviewed person.\n\n" + emailSignature, { cc: managerEmail });        //row.getCell(1,10).setValue("Email Draft");
```

Comment:
> //row.getCell(1,10).setValue("Email Draft");

@yanjoseph seems something unintended? Should you drop it?

Reply:
removed the redundant comment